### ISSUE TITLE
[request] missing option :leak-detection-threshold

### ISSUE THREAD

**Issue by timgluz** [tomekw/hikari-cp#18]
Hi,

i'm currently trying to find a connection leak in my app and i noticed i cant change [leakDetectionThreshold](https://github.com/brettwooldridge/HikariCP#infrequently-used) value.

**Comment by jstepien** [comment]
Hi @timgluz,

Would it work for you to set it [using the `configure` option][cfg]? It would probably something along the lines of:

```clojure
(make-datasource {:configure (fn [cfg] (.setLeakDetectionThreshold ...))})
```

[cfg]: https://github.com/tomekw/hikari-cp/blob/de94afc11c82fb8333bb11b526dbe253071b5dc8/src/hikari_cp/core.clj#L132

**Comment by timgluz** [comment]
i'll give a try - but i already forked a project to add a new optional-key into settings map.

**Comment by jstepien** [comment]
In such case a PR is more than welcome! I'd be excellent if your patch also updates the list of options in README and adds a changelog entry.

**Comment by timgluz** [comment]
@jstepien Btw, do you use hikari-cp in production? Have you ever encountered connection-leak with it?

**Comment by jstepien** [comment]
We do, with both MySQL and PostgreSQL, and to the best of my knowledge we haven't experienced problems you described in https://github.com/brettwooldridge/HikariCP/issues/513

**Comment by jstepien** [comment]
Merged in  #19
